frontend/src/components/events/molecules/event_card.css

```diff
@@ -0,0 +1,14 @@
+.eventCard {
+  border-radius: 5px;
+  padding: 5px;
+  transition: box-shadow 0.3s;
+  max-width: 400px;
+}
+
+.eventCard:hover {
+  box-shadow: 0 0 5px rgba(33, 33, 33, 0.8);
+}
+
+.col {
```

Comment:
this is causing a problem on the home page now. make sure you use a unique class name to solve this issue

Reply:
Gave column text box a unique name to resolve conflict. Pushed up changes.